### ISSUE TITLE
FusedBatchNormV3 with empty var conversion looks incorrect

### ISSUE THREAD

**Issue by saurabh-shandilya** [onnx/tensorflow-onnx#1075]
**Describe the bug**
The tensorflow op  FusedBatchNormV3 with empty mean and var variables when converted leads to mean and var with values zero. While mean being zero is fine, but var being 0 might lead to division by a really small epsilon. I think the ideal behavior should have been to keep the variance as (1-epsilon). 

**Urgency**

**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Windows
- Tensorflow Version: tf1.15
- Python version: 3.6
- tf2onnx version: 1.63

**To Reproduce**
I can't share the model, but following command was used - 
python -m tf2onnx.convert --input frozen_model.pb --fold_const --opset 10 --inputs Placeholder:0[1,704,960,3] --outputs logits/Conv2D:0 --output converted.onnx

( happens without --fold_const option, when fold_const option is used, during Conv/BN merging it seems to be handled correctly) 

**Expected behavior**

In FusedBatchNormV3 if var is empty, it should be made [1-epsilon] (to cancel the effect of epsilon) instead of 0.  

**Screenshots**
TF Model
![tf](https://user-images.githubusercontent.com/40773275/90929610-e93a9780-e3ad-11ea-8f52-78c86323ad7f.jpg)
After conversion
![converted](https://user-images.githubusercontent.com/40773275/90929614-eb045b00-e3ad-11ea-933c-5ac191519640.jpg)

**Additional context**
Add any other context about the problem here. If the issue is about a particular model, please share the model details as well to facilitate debugging.
